src/components/transverse/EnTete/EnTete.tsx

```diff
@@ -95,27 +99,29 @@ export default function EnTete(): ReactElement {
                       href="/"
                     >
                       Notifications
-                      {/**/}
+                      {' '}
```

Comment:
```suggestion
                      {/**/}
```
C'était correct avant sinon ça ajoute une espace.

Reply:
Fait.